Marlin/planner.h

```diff
@@ -395,10 +395,16 @@ class Planner {
 
     #if ENABLED(ULTRA_LCD)
 
-      static millis_t block_buffer_runtime() {
+      static uint16_t block_buffer_runtime() {
         CRITICAL_SECTION_START
           millis_t bbru = block_buffer_runtime_us;
         CRITICAL_SECTION_END
+        // To translate µs to ms a division by 1000 would be required.
+        // We introduce 2.4% error her by dividing by 1024.
+        // Does not matter because block_buffer_runtime_us is already an, too small, estimation.
```

Comment:
It really does not matter. All we need here is 'about' milliseconds.

When we use this value we compare with its half. Even a unexactness if +-20% would not have a bad consequence. If so it would be better to change the formula on the other side - when we are comparing. Dividing by 1 would allow to use up up to all the planner buffer content for the display refresh. Two is what we have. Tree would allow to use up to 1/3 of the buffer content - is also a reasonable value - would just stop display updates a bit more frequent. But multiplying by 3 is again a bit slower than just shifting. (Does it really matter - only 13 times calculated per second. Here we calculate with 16 bits only)